libs/net/dal/Migrations/1.0.59/Up/PostUp/00-Topic.sql

```diff
@@ -0,0 +1,524 @@
+DO $$
```

Comment:
Change your DB migration to `1.0.60`

Run the following commands to ensure you have done it correctly.
` cd libs/net` folder
`make db-rollback n=1.0.58` - Undo your migration
`make db-remove` - Remove all generated code in your migration
Move your scripts to `1.0.60`
Get the latest and update your branch with a rebase
`make db-update` - Update to the latest
`make db-add n=1.0.60` - Add your migration
`make db-update` - Update the database
Now test rollback and update again
`make db-rollback n=1.0.59`
`make db-update`

Reply:
Please disregard this PR as I just submitted another PR for it. Thanks